components/inputs/input-text.js

```diff
@@ -464,10 +473,11 @@ class InputText extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))) {
 
 	_handleFocus() {
 		this._focused = true;
+		this._focusedOrHovered = true;
 	}
 
 	_handleInput(e) {
-		this.value = e.target.value;
+		this._value = e.target.value; // set directly to avoid re-render which causes cursor jumping in Safari
```

Comment:
This bypasses the logic in `updated()` for validation/form stuff... working on a better fix.